SPECS-EXTENDED/python-nose2/python-nose2.spec

```diff
@@ -0,0 +1,183 @@
+BuildRequires:  python%{python3_other_pkgversion}-mock
+BuildRequires:  python%{python3_other_pkgversion}-cov-core
+%endif
+
+%description
+nose2 is the next generation of nicer testing for Python, based on the plugins
+branch of unittest2. nose2 aims to improve on nose by:
+- providing a better plugin API
+- being easier for users to configure
+- simplifying internal interfaces and processes
+- supporting Python 2 and 3 from the same codebase, without translation
+- encouraging greater community involvement in its development
+
+In service of some those goals, some features of nose will not be supported in
+nose2. See the documentation for a thorough rundown.
+
+
+%package -n python%{python3_pkgversion}-%{srcname}
+Summary:        Next generation of nicer testing for Python
+%{?python_provide:%python_provide python%{python3_pkgversion}-%{srcname}}
+Requires:       python%{python3_pkgversion}-setuptools
+Requires:       python%{python3_pkgversion}-six >= 1.6
+Requires:       python%{python3_pkgversion}-cov-core >= 1.12
+Conflicts:      python2-%{srcname} < 0.7.4-3
+
+%description -n python%{python3_pkgversion}-%{srcname}
+nose2 is the next generation of nicer testing for Python, based on the plugins
+branch of unittest2. nose2 aims to improve on nose by:
+- providing a better plugin API
+- being easier for users to configure
+- simplifying internal interfaces and processes
+- supporting Python 2 and 3 from the same codebase, without translation
+- encouraging greater community involvement in its development
+
+In service of some those goals, some features of nose will not be supported in
+nose2. See the documentation for a thorough rundown.
+
+
+%if 0%{?with_python3_other}
+%package -n python%{python3_other_pkgversion}-%{srcname}
+Summary:        Next generation of nicer testing for Python
+%{?python_provide:%python_provide python%{python3_other_pkgversion}-%{srcname}}
+Requires:       python%{python3_other_pkgversion}-setuptools
+Requires:       python%{python3_other_pkgversion}-six >= 1.6
+Requires:       python%{python3_other_pkgversion}-cov-core >= 1.12
+
+%description -n python%{python3_other_pkgversion}-%{srcname}
+nose2 is the next generation of nicer testing for Python, based on the plugins
+branch of unittest2. nose2 aims to improve on nose by:
+- providing a better plugin API
+- being easier for users to configure
+- simplifying internal interfaces and processes
+- supporting Python 2 and 3 from the same codebase, without translation
+- encouraging greater community involvement in its development
+
+In service of some those goals, some features of nose will not be supported in
+nose2. See the documentation for a thorough rundown.
+%endif
+
+
+%prep
+%autosetup -n %{srcname}-%{version} -p0
+
+
+%build
+%py3_build
+%if 0%{?with_python3_other}
+%py3_other_build
+%endif
+
+
+%install
+# Must do the default install last because the scripts in /usr/bin are
+# overwritten with every setup.py install.
+%if 0%{?with_python3_other}
```

Comment:
We don't really have a need to support multiple python3 versions in Mariner- can we remove this construct?